bundles/org.openhab.binding.modbus.sunspec/README.md

```diff
@@ -0,0 +1,297 @@
+
+You can add `enableDiscovery=true` attribute to your bridge config, or you can enable it in the paper ui under the modbus tcp|serial slave thing.
+
+A typical bridge configuration would looke like this:
+
+```
+Bridge modbus:tcp:bridge [ host="10.0.0.2", port=502, id=1, enableDiscovery=true ]
+```
+
+### Adding things manually
+
+If you decide to add a thing manually then first you have to find out the start address of the model block and the length of it.
+While the length is usually fixed the address isn't.
+Please refer to your device's vendor documentation how model blocks are laid for your equipment.
+
+The following parameters are valid for all thing types:
+
+| Parameter | Type    | Required | Default if ommitted | Description                             |
+|-----------|---------|----------|---------------------|-----------------------------------------|
+| address   | integer | yes      | N/A                 | Start address of the model block.       |
+| length    | integer | yes      | N/A                 | Length of the model block. Setting this too short could cause problems during parsing |
+| refresh   | integer | no       | 5                   | Poll inteval in seconds. Increase this if you encounter connection errors |
+| maxTries  | integer | no       | 3                   | Number of retries when before giving up reading from this thing. |
+
+
+## Channels
+
+Channels are grouped into channel groups.
+Different things support a subset of the following groups.
+
+### Device information group (deviceInformation)
+
+This group contains general operational information about the device.
+
+| Channel ID           | Item Type | Description                                                         |
+|----------------------|-----------|---------------------------------------------------------------------|
+| cabinet-temperature  | Number    | Temperature of the cabinet if supported in Celsius                  |
+| heatsink-temperature | Number    | Device heat sink temperature in Celsius                             |
+| transformer-temperature | Number    | Temperature of the transformer in Celsius                        |
+| other-temperature    | Number    | Any other temperature reading not covered by the above items if available. Celsius |
+| status               | Number    | Device status: 1=Off, 2=Sleeping/night mode, 4=On - producing power |
+
+Supported by: all inverter things
+
+### AC summary group (acGeneral)
+
+#### inverters
+
+This group contains summarized values for the AC side of the inverter.
+Even if the inverter supports multiple phases this group will appear only once.
+
+| Channel ID           | Item Type | Description                                                         |
+|----------------------|-----------|---------------------------------------------------------------------|
+| ac-total-current     | Number    | Total AC current over all phases in Amperes                         |
```

Comment:
```suggestion
| ac-total-current     | Number:ElectricCurrent    | Total AC current over all phases in Amperes                         |
```

also adjust the other channels to the correct type

Reply:
ok, adjusted the channel types